torchtune/models/clip/_component_builders.py

```diff
@@ -99,3 +127,12 @@ def clip_vision_encoder(
         embed_dim=embed_dim,
         in_channels=in_channels,
     )
+
+
+def clip_mlp(in_dim: int, out_dim: int, hidden_dim: int, activation: nn.Module, quantize_base: bool = False) -> FeedForward:
+    """
+    Build the MLP layer associated with the clip model.
```

Comment:
Not *the* CLIP model, right? I feel like we can be a bit more explicit here

Reply:
You mean CLIP ViT? I feel this is inline with naming for all our other model specific sub builders.